sysboundary/sysboundary.cpp

```diff
@@ -625,78 +661,80 @@ bool SysBoundary::applyInitialState(
  *
  * Loops through all SysBoundaryConditions and calls the corresponding vlasovBoundaryCondition() function.
  * The boundary condition functions are called for all particle species, one at a time.
- * 
- * WARNING (see end of the function) Blocks are changed but lists not updated now, 
- * if you need to use/communicate them before the next update is done, add an 
+ *
+ * WARNING (see end of the function) Blocks are changed but lists not updated now,
+ * if you need to use/communicate them before the next update is done, add an
  * update at the end of this function.
- * 
+ *
  * \param mpiGrid Grid
  * \param t Current time
  */
 void SysBoundary::applySysBoundaryVlasovConditions(
-   dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid,
-   creal& t,
-   const bool calculate_V_moments // if true, compute into _V, false into _R moments so that the interpolated ones can be done
+    dccrg::Dccrg<SpatialCell, dccrg::Cartesian_Geometry>& mpiGrid, creal& t,
+    const bool
+        calculate_V_moments // if true, compute into _V, false into _R moments so that the interpolated ones can be done
 ) {
-   if(sysBoundaries.size()==0) {
-      return; //no system boundaries
+   if (sysBoundaries.size() == 0) {
+      return; // no system boundaries
    }
 
-   /*Transfer along boundaries*/
-   // First the small stuff without overlapping in an extended neighbourhood:
-   #warning TODO This now communicates in the wider neighbourhood for both layers, could be reduced to smaller neighbourhood for layer 1, larger neighbourhood for layer 2.
+/*Transfer along boundaries*/
+// First the small stuff without overlapping in an extended neighbourhood:
+#warning TODO This now communicates in the wider neighbourhood for both layers, could be reduced to smaller neighbourhood for layer 1, larger neighbourhood for layer 2.
    SpatialCell::set_mpi_transfer_type(
-      Transfer::CELL_PARAMETERS|
-      Transfer::POP_METADATA|
-      Transfer::CELL_SYSBOUNDARYFLAG,true);
+       Transfer::CELL_PARAMETERS | Transfer::POP_METADATA | Transfer::CELL_SYSBOUNDARYFLAG, true);
```

Comment:
Either all indented or all in one line, but just all arguments on another line is even weirder than what was before I think.